lib/preview-web/src/PreviewWeb.mockdata.ts

```diff
@@ -0,0 +1,97 @@
+import { EventEmitter } from 'events';
+import Events from '@storybook/core-events';
+import { StoriesList } from '@storybook/store';
```

Comment:
```suggestion
import { StoryIndex } from '@storybook/store';
```

I think we need a way to typecheck tests

Reply:
Yeah 🤷